src/Lib/Managers/ItemsManager.vala

```diff
@@ -202,7 +207,11 @@ public class Akira.Lib.Managers.ItemsManager : Object {
 
     private void udpate_default_values () {
         border_size = settings.set_border ? settings.border_size : 0.0;
-        border_color = settings.set_border ? settings.border_color: "";
-        fill_color = settings.fill_color;
+
+        string border_color_str = settings.set_border ? settings.border_color : DEFAUL_BORDER_COLOR;
+        string fill_color_str = settings.fill_color;
+
+        border_color.parse (border_color_str);
+        fill_color.parse (fill_color_str);
```

Comment:
No need to define the `fill_color_str` variable if there's no logic or condition attached to it.
Just type hint it when you parse it:
`fill_color.parse ((string) settings.fill_color);`